api_tests/lib/bitcoin.js

```diff
@@ -47,38 +48,30 @@ module.exports.btc_activate_segwit = async function() {
     return create_bitcoin_rpc_client().generate(432);
 };
 
-module.exports.btc_import_address = async function(address) {
-    return create_bitcoin_rpc_client().importAddress(address);
-};
+async function getSatoshiTransferredTo(tx_id, address) {
```

Comment:
❌ This is a halfway implementation. To do what this says it should iterate over all utxos going to the address and add up the total.

Reply:
It is intentional, because we only want to consider transactions that have one utxo, built using the action that we return in the HTTP API.